### ISSUE TITLE
ignore notes that contain specific tag/s

### ISSUE THREAD

**Issue by DryIce1** [dvcrn/obsidian-filename-heading-sync#59]
Hello,

thanks for this plugin, it is fantastic!

- **Problem**: I don't know how to ignore files that contain tag `#dailynotes`.
- **Use case**: some note types (such as my daily notes) never require an `# H1` and the Filename Heading Sync inappropriately adds an `# H1`. These files usually have a tag that distinguishes them from other types of notes.
- **existing capability:** The RegEx ignore rule only acts upon folders to ignore files, #12. But the limitation is that it doesn't search the text within a note and ignore tags.
- **Proposed solution**: 
    1. Add support to ignore specific tag/s.
    2. Ignore file when metadata contains field, such as `filename-heading-sync: false`, as seen in [the embedded note titles plugin](https://github.com/mgmeyers/obsidian-embedded-note-titles/issues/14#issuecomment-992911568]

The ignore specific tags is preferred, because it uses tag metadata which already exists within the note.

thanks for considering or directing me to an already existing solution. 👍

**Comment by dvcrn** [comment]
Yeah we should add something like this. Both filecontent ignores and metatags sound useful

**Comment by doitian** [comment]
I found a workaround: puting the filename heading in the comment. Both the markdown comment and HTML comment works.

Example:

```
%%
# Filename
%%
# My Title
```

or 

```
<!--
# Filename
-->
# My Title
```

Now the filename is synced with the heading in the comment, where the real title is not affected.

**Comment by dvcrn** [comment]
Why not just ignore the file from this plugin if you don't want the heading to be synced? Like, why have it sync to a heading in comments?

**Comment by doitian** [comment]
Some edge cases to ignore the plugin in files which has no particular path pattern. For example, I often extract a chapter from a main file but want to keep the non-h1 chapter title.

**Comment by jerrygreen** [comment]
I would like to ignore not just a tag but an entire folder. Unfortunately, the extension doesn't have settings for that.

I don't like my daily notes having a title, or tags, or anything except my text. Would like to keep them as simple as possible.
